Library/Homebrew/test/caveats_spec.rb

```diff
@@ -91,7 +91,7 @@ def plist
           expect(caveats).to include("restart #{f.full_name} after an upgrade")
         end
 
-        it "tells command to run to start formula" do
+        it "tells command to run to start formula", :needs_macos do
```

Comment:
How about adding this to all the plist tests? You can also add this to `context` and `describe` calls to skip large sets of tests.